### PR TITLE
adds task describe command

### PR DESCRIPTION
**Pull request by jritsema**
```console
fargate task describe
```

The describe command describes a [Task Definition](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html) in [Docker Compose](https://docs.docker.com/compose/overview/) format. The Docker image, environment variables, secrets, and target port are the mapped elements.

This command can be useful for looking at changes made by the `task register`, `service deploy`, or `fargate service env set` commands.  It can also be useful for running a task definition locally for debugging or troubleshooting purposes.

```sh
fargate task describe -t my-app > docker-compose.yml
docker-compose up
```

You can specify the task definition family by using a `fargate.yml` file, the `FARGATE_TASK` envvar, or using 
the `-t` flag, including an optional revision number.

```sh
fargate task describe -t my-app
fargate task describe -t my-app:42
```

Example output:

```yaml
version: "3.7"
services:
  app:
    image: 1234567890.dkr.ecr.us-east-1.amazonaws.com/my-app:1.0
    ports:
    - published: 8080
      target: 8080
    environment:
      AWS_REGION: us-east-1
      ENVIRONMENT: dev
      FOO: bar
    x-fargate-secrets:
      KEY: arn:key:ssm:us-east-1:000000000000:parameter/path/to/my_parameter      
    labels:
      aws.ecs.fargate.deploy: "1"
```

### CODE REVIEW

File: cmd/task_generate.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+	//write object to file
+	yes := true
+	if _, err := os.Stat(composeFile.File); err == nil {
+		fmt.Print(composeFile.File + " already exists. Overwrite? ")
+		yes = askForConfirmation()
+	}
+	if yes {
+		composeFile.Write()
+		fmt.Println("wrote", composeFile.File)
```
Comment (abarfoot) [inline cmd/task_generate.go]: %s ?

Comment (jritsema) [reply cmd/task_generate.go]: > For each Printf-like function, there is also a Print function that takes no format and is equivalent to saying %v for every operand. Another variant Println inserts blanks between operands and appends a newline.

https://golang.org/pkg/fmt/

**Comment by abarfoot** [comment]
Can you think of an alternative to `generate`?  It's not obvious that `fargate task generate` generates a docker compose file... sounds like it would somehow generate a task.

**Comment by jritsema** [comment]
> Can you think of an alternative to `generate`? It's not obvious that `fargate task generate` generates a docker compose file... sounds like it would somehow generate a task.

Good question.  I went with the equivalent command in harbor-compose.  But I'd be open to alternatives.

**Comment by ajliv** [comment]
I have no issue with `generate`. Alternatives I can think of would be `scaffold` or `describe`.

Maybe `describe` would simply echo the output then something like this could be run
```bash
fargate task describe > docker-compose.yml
```

**Comment by jritsema** [comment]
I like `describe`.  I also like just outputting it to stdout.

**Pull request merged**